mobile/build.gradle

```diff
@@ -168,6 +171,7 @@ dependencies {
     implementation "com.mikepenz:aboutlibraries-core:$about_libraries_version"
     implementation "com.mikepenz:aboutlibraries:$about_libraries_version"
     implementation "com.github.GrenderG:Toasty:1.5.2"
+    implementation "org.mozilla.geckoview:geckoview-beta:96.0.20211219185855"
```

Comment:
Suggestion: Switch to stable channel to prevent developing against features that are not available yet